pipewrench/merge.py

```diff
@@ -348,7 +348,12 @@ def sqoop_map_java_column(columns, clean_column=False):
             column_name = cleanse_column(column_name)
 
         datatype = column['datatype'].lower()
-        if datatype in ['clob', 'longvarbinary', 'varbinary', 'rowid', 'blob',
+        if 'map_java_column' in column:
+            override = column['map_java_column']
+            logging.debug('Override datatype for %s to %s', column_name, override)
+            mapped_columns = True
+            map_java_column = map_java_column + "'{name}=".format(name=column_name) + "{type}".format(type=override) + "',"
+        elif datatype in ['clob', 'longvarbinary', 'varbinary', 'rowid', 'blob',
                         'nclob', 'text', 'binary']:
```

Comment:
this line needs two more spaces - align the single quote. build is failing due to linting.